### ISSUE TITLE
Loans from and for Players

### ISSUE THREAD

**Issue by MaggiMiner** [jeremyckahn/farmhand#29]
make it possible to give out loans to other players, and a rate the loan giver wants, with that new players get money, and rich players get passive income. This could be an addition or replacement of the current Loan system

**Comment by jeremyckahn** [comment]
Hah! I love this idea. It’s definitely an interesting take on expanding the multiplayer features.

This data will have be stored in a non-volatile data store (like MongoDB or Postgres). Right now all we have is a Redis instance, which is volatile. This will have to be a longer term feature, but I will keep it open!

**Comment by MaggiMiner** [comment]
for the loans, if a player restarts the save, the person(s) giving out the loan(s) should be refunded.

**Comment by karlerik-simon** [comment]
Sound like a fun issue!

**Comment by jeremyckahn** [comment]
> Sounds like a fun issue!

Agreed! As I mentioned above, this would require some backend infrastructure that we don't have now. We might be able to get something basic working with free tiers for some services like MongoDB, but eventually we'll have to bite the bullet and look to using Patreon or something to fund proper server resources.

**Comment by karlerik-simon** [comment]
Indeed, I could try setting something up and just playing around with different alternatives.

**Comment by jeremyckahn** [comment]
@karlerik-simon please do! I don't have a firm idea about how this all might work, and I'd like to see what you and others have in mind.
